index.php

```diff
@@ -11,5 +11,19 @@
     echo json_encode($datas);
 } catch (\Exception $e) {
     header("HTTP/1.0 500 Internal Server Error");
-    echo $e->getMessage();
-}
\ No newline at end of file
+    if ($e instanceof PDOException) {
+        echo 'PBM de DB : ' . $e->getMessage();
```

Comment:
Why catch PDoExcption? It is already catch in adapter

Reply:
done